frontend/src/metabase-lib/v1/expressions/complete/index.ts

```diff
@@ -1 +1,3 @@
 export * from "./complete";
+export { type Shortcut } from "./types";
+export { type Completion } from "./types";
```

Comment:
Help and dropdown are displayed simultaneously. Is this expected?

To reproduce type: `coalesce(123, d`

![image](https://github.com/user-attachments/assets/506e5211-73e9-4415-887c-0250b68782ba)

Reply:
Yes! This is the goal of the new popover.